app/experimenter/templates/experiments/section_overview.html

```diff
@@ -9,6 +9,8 @@
   {% if experiment.engineering_owner %}
     <p><strong>Engineering Owner:</strong> {{ experiment.engineering_owner }}</p>
   {% endif %}
+  <p><strong>Public Name:</strong> {{ experiment.public_name }}</p>
+  <p><strong>Public Description:</strong> {{ experiment.public_description }}</p>
```

Comment:
Ah you have to add a [linebreaks tag](https://docs.djangoproject.com/en/2.1/ref/templates/builtins/#linebreaks) here